wordpress-mobile/WordPress-Android
# Fix crash starting GB on new posts

Fixes #9099 by using `setContent` instead of `initContent` when initializing the bridge code.

To test:
- Start the app
- Tap on the New post main button
- It should work fine


Companion PR gutenberg-mobile side is here: https://github.com/wordpress-mobile/gutenberg-mobile/pull/496

Update release notes:
